src/message.rs

```diff
@@ -1762,6 +1768,9 @@ pub enum Viewtype {
     /// The text of the message is set using dc_msg_set_text() and retrieved with dc_msg_get_text().
     Text = 10,
 
+    /// RFC 9078 reaction.
+    Reaction = 11,
```

Comment:
i did not expect that to be exposed to the viewtype - similar as there is no DC_MSG_REACTION and MDN do not have a viewtype.

is this needed to make some code simpler, so a pragmatic approach?

Reply:
i also played around with the repl tool, where you can already send a reaction with eg, `react MSGID ❤️` - i did not expect these reaction to appear as normal messages in the repl tool (they appear as normal messages in clients that do not use this pr, this is fine).